www/js/cci-about/about.js

```diff
@@ -19,5 +24,7 @@ angular.module('emission.main.cci-about', ['emission.plugin.logger'])
            window.Logger.log(window.Logger.LEVEL_INFO,
                "CCI email cancel reported, seems to be an error on android");
         });
-    };
+  };
+  var SURVEY_DONE_KEY = 'survey_done';
```

Comment:
you only need this if you want the survey to pop-up automatically. And then you would set the `SURVEY_DONE_KEY` to true once the user finished the survey. I commented this out for the goals screen since we didn't want to popup a survey.

Original code for the popup is here:
https://github.com/e-mission/e-mission-phone/commit/6d86ce5fb6ed3dfa385b19e02ea9d574e2345755#diff-c3e554e73846808d6fb5cf0538e649c5 (line 933 onwards)